kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -320,6 +340,36 @@ public String toString() {
     }
   }
 
+  private static class CannedAclValidator implements ConfigDef.Validator {
+    public static Map<String, CannedAccessControlList> aclsByHeaderValue = new HashMap<>();
```

Comment:
Cool. Final brush strokes, let's make those two `final` and then comply with the format for `static final` member fields. That would be `ACLS_BY_HEADER_VALUE` and `ALLOWED_VALUES` respectively.